lms/templates/learner_dashboard/course_card.underscore

```diff
@@ -1,25 +1,27 @@
 <div class="section">
     <div class="course-meta-container">
-        <div class="course-details">
```

Comment:
This file has nothing to do with registration. It should not have been modified.